src/components/profile/profile.css

```diff
@@ -1,8 +1,33 @@
 .logout-container {
   border-radius: 5px;
+  border: 1px solid var(--default-color);
   height: fit-content;
-  margin: 1rem;
+  margin-left: 7rem;
+  margin-top: 3rem;
   max-width: 30rem;
-  padding: 6rem;
-  row-gap: 0.5rem;
+  padding: 2rem;
+  row-gap: 1rem;
+}
+
+.row-format {
+  column-gap: 2rem;
+  display: flex;
+}
+
+.profile-column {
+  row-gap: 1rem;
+}
+.profile-details {
+  column-gap: 2rem;
+}
+
+.logout-btn {
+  margin-left: 0;
+  margin-top: 0;
```

Comment:
You can use shorthand property here

Reply:
Here, I have to keep the margin top and bottom as it is, thats why not assigning the value again